vm/src/thread.rs

```diff
@@ -241,6 +241,7 @@ impl RootedThread {
                 stack: Stack::new(),
                 record_map: FieldMap::new(),
                 hook: None,
+                stack_size_limit: VmIndex::max_value(),
```

Comment:
Wanted to distinguish the maximum total allowed stack size from the the maximum required for a specific function, but I may have failed in that respect :/.